src/EventBuilder/Platforms/Essentials.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NuGet;
+using Polly;
+using Serilog;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace EventBuilder.Platforms
+{
+    public class Essentials : BasePlatform
+    {
+        private const string _packageName = "Xamarin.Essentials";
+
+        public override AutoPlatform Platform => AutoPlatform.Essentials;
+
+        public Essentials()
+        {
+            var packageUnzipPath = Environment.CurrentDirectory;
+
+            var retryPolicy = Policy
+                .Handle<Exception>()
+                .WaitAndRetry(
+                    5,
+                    retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)),
+                    (exception, timeSpan, context) =>
+                    {
+                        Log.Warning(
+                            "An exception was thrown whilst retrieving or installing {packageName}: {exception}",
+                            _packageName, exception);
+                    });
+
+            retryPolicy.Execute(() =>
+            {
+                var repo = PackageRepositoryFactory.Default.CreateRepository("https://packages.nuget.org/api/v2");
+                var packageManager = new PackageManager(repo, packageUnzipPath);                
+                var fpid = packageManager.SourceRepository.FindPackagesById(_packageName);
+                var package = fpid.Single(x => x.Version.ToString() == "0.8.0-preview");
+
+                packageManager.InstallPackage(package, true, true);
+
+                Log.Debug("Using Xamarin Essentials {Version} released on {Published}", package.Version, package.Published);
+                Log.Debug("{ReleaseNotes}", package.ReleaseNotes);
+            });
+
+            var xamarinForms =
+                Directory.GetFiles(packageUnzipPath,
+                    "Xamarin.Essentials.dll", SearchOption.AllDirectories);
+
+            var latestVersion = xamarinForms.Last();//x => x.Contains("netstandard2.0"));
```

Comment:
is the x.contains needed?

Reply:
Originally I had it pinned at `netstandard2.0`, but then had to switch it to try to just get the last one. I am going to pin it at `netstandard2.0`.